model/src/test/kotlin/config/OrtConfigurationTest.kt

```diff
@@ -69,7 +69,7 @@ class OrtConfigurationTest : WordSpec({
                 }
 
                 storages shouldNotBeNull {
-                    keys shouldContainExactlyInAnyOrder setOf("local", "http", "clearlyDefined", "postgres")
+                    keys shouldContainExactlyInAnyOrder setOf("local", "http", "clearlyDefined", "postgres", "sw360")
```

Comment:
It's "sw360Configuration" instead of "sw360".

Reply:
Fixed